<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDialog</class>
 <widget class="QDialog" name="ImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Contraste</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Brillo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Saturaci√≥n</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSlider" name="barraSaturacion">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSlider" name="barraContraste">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSlider" name="barraBrillo">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>barraBrillo</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ImageDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barraContraste</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ImageDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barraSaturacion</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ImageDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAceptar</sender>
   <signal>pressed()</signal>
   <receiver>ImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
